Description: update paths for lz4_decompress and zlib_decompress
 Since these binaries are moved to /usr/lib/mysql/ by packaging, we also
 need to ensure that the tests can find them.
Author: Robie Basak <robie.basak@canonical.com>
Forwarded: not-needed
Last-Update: 2016-03-17

Index: mysql-5.7.15/mysql-test/mysql-test-run.pl
===================================================================
--- mysql-5.7.15.orig/mysql-test/mysql-test-run.pl	2016-09-12 10:36:28.138747572 -0400
+++ mysql-5.7.15/mysql-test/mysql-test-run.pl	2016-09-12 10:36:28.138747572 -0400
@@ -2737,7 +2737,7 @@
   # lz4_decompress
   # ----------------------------------------------------
   my $exe_lz4_decompress= mtr_exe_exists(vs_config_dirs('extra', 'lz4_decompress'),
-                                 "$basedir/extra/lz4_decompress",
+                                 "/usr/lib/mysql/lz4_decompress",
                                  "$path_client_bindir/lz4_decompress");
   $ENV{'LZ4_DECOMPRESS'}= native_path($exe_lz4_decompress);
 
@@ -2745,7 +2745,7 @@
   # zlib_decompress
   # ----------------------------------------------------
   my $exe_zlib_decompress= mtr_exe_exists(vs_config_dirs('extra', 'zlib_decompress'),
-                                 "$basedir/extra/zlib_decompress",
+                                 "/usr/lib/mysql/zlib_decompress",
                                  "$path_client_bindir/zlib_decompress");
   $ENV{'ZLIB_DECOMPRESS'}= native_path($exe_zlib_decompress);
 
